### PR TITLE
enable spelling checks on blog posts

### PR DESCRIPTION
**Pull request by oraNod**
First we need to merge this PR so that we don't try to spell check the bullhorn archive: https://github.com/ansible-community/community-website/pull/407

### CODE REVIEW

**Comment by webknjaz** [comment]
@oraNod also consider integrating https://vale.sh — apparently, it also has a Red Hat style: https://vale.sh/hub/redhat/ / https://redhat-documentation.github.io/vale-at-red-hat/docs/main/user-guide/redhat-style-for-vale/. I'm eager to try it myself but haven't yet, though.

**Review by GregSutcliffe — DISMISSED** [review]
Assuming it passes tests after 407 is merged, LGTM. I like the idea from @webknjaz, but let's iterate rather than blocking this PR.

**Comment by oraNod** [comment]
Thanks @GregSutcliffe the spelling check should pass now that I've rebased. The force push resets your approval but I'll merge anyway.

Agree that the vale linter is an interesting suggestion and I appreciate the shout @webknjaz I'm quite familiar with it actually. I wonder if some of those Red Hat rules might be a bit opinionated for a community website honestly, but it's certainly worth discussion. On it's own though vale is great. I've toyed around with some custom rules that help avoid complexity in a doc set. Maybe something to consider for the ansible-documentation repo.

**Pull request merged**
